content/docs/user-guide/data-pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,237 @@
+# Defining Pipelines
+
+DVC pipelines are written as collections of data processing <abbr>stages</abbr>
+in `dvc.yaml` files. These represent data workflows that you want to
+**reproduce** reliably later. Here's a sample 3-stage structure:
+
+```yaml
+stages:
+  prepare: ... # stage 1 definition
+  train: ... # stage 2 definition
+  evaluate: ... # stage 3 definition
+```
+
+<admon>
+
+This _codification_ has the added benefit of allowing you to develop pipelines
+on standard Git workflows (GitOps).
+
+</admon>
+
+A pipeline is formed by making [stages](#stages) interdependent, meaning that
```

Comment:
Considering the next section it feel redundant?

Reply:
No, I think they're complementary: this text (now updated) states how pipelines are formed (by connecting stages), the DAG section is more conceptual and explains how pipelines are represented internally (as graphs of the aforementioned stage dependencies).